guides/v2.1/frontend-dev-guide/translations/xlate.md

```diff
@@ -84,6 +84,18 @@ Only one variant of translation can be used for a word or phrase in a package. O
 
 Anyone can submit inline translations on the {% glossarytooltip 1a70d3ac-6bd9-475a-8937-5f80ca785c14 %}storefront{% endglossarytooltip %} using the Text Editor. These inline translations overwrite a dictionary and save to your database (not in an instance's dictionary). Inline translations are theme-specific and do not apply if another theme is assigned.
 
+## How inline translation works
+
+The preferred way of translating the Storefront in Magento is by using the translation dictionaries (.csv files) and language packages in the Magento codebase.
```

Comment:
This seems extraneous. Consider moving to the end of this section and prepending something like: 

> Note: inline translation is not intended to be used as the primary localization technique in Magento. Instead, the preferred way of translating is by using the translation...